src/Debug/Pretty/Simple.hs

```diff
@@ -228,7 +246,83 @@ pTraceMarker = traceMarker . unpack . pString
 --
 -- @since 2.0.1.0
 pTraceMarkerIO :: String -> IO ()
-pTraceMarkerIO = traceMarkerIO . unpack . pString
+pTraceMarkerIO = traceMarkerIO . unpack <=< pStringTTYIO
+
+------------------------------------------
+-- Helpers
+------------------------------------------
+
+pStringTTYIO :: String -> IO Text
```

Comment:
I don't like it but it works, and I couldn't find a better solution quickly, and I wanted to push something out before going to sleep lol

Reply:
This seems reasonable.  I changed it to check `stderr` instead of `stdout`, since it looks like the `trace` functions output to `stderr`.  But other than that this looks good.